extensions/amp-subscriptions/0.1/subscription-platform.js

```diff
@@ -57,10 +57,11 @@ export class SubscriptionPlatform {
   pingback(unusedSelectedPlatform) {}
 
   /**
-   * Tells if this platform supports the current viewer.
-   * @return {boolean}
+   * Tells if the platform supports a score factor
+   * @param {string} unusedFactor
+   * @return {!Promise<number>}
    */
-  supportsCurrentViewer() {}
+  getSupportedFactor(unusedFactor) {}
```

Comment:
maybe rename this to "getSupportedScoreFactor" to make this method more clear (also in other places it is used)